include/crm/common/resources.h

```diff
@@ -27,6 +27,7 @@ enum pe_obj_types {
     pcmk_rsc_variant_primitive  = 0,    //!< Primitive resource
     pcmk_rsc_variant_group      = 1,    //!< Group resource
     pcmk_rsc_variant_clone      = 2,    //!< Clone resource
+    pcmk_rsc_variant_bundle     = 3,    //!< Bundle resource
```

Comment:
for consistency we might use that for pe_container below - but well ...

Reply:
whoops, i meant to